### ISSUE TITLE
Ognl.createDefaultContext(Object, ClassResolver) throws "IllegalArgumentException"

### ISSUE THREAD

**Issue by oosato-im** [orphan-oss/ognl#162]
Ognl.createDefaultContext(Object, ClassResolver) always throws IllegalArgumentException, because "memberAccess" of OgnlContext Constructor is set to null.
"memberAccess" would be set to memberAccess similar to Ognl.createDefaultContext(Object).

https://github.com/jkuhnert/ognl/blob/OGNL_3_3_3/src/main/java/ognl/Ognl.java#L255
https://github.com/jkuhnert/ognl/blob/OGNL_3_3_3/src/main/java/ognl/OgnlContext.java#L144

Ognl version is 3.3.3.

**Comment by lukaszlenart** [comment]
As far I see a default `MemberAccess` is [created](https://github.com/orphan-oss/ognl/blob/OGNL_3_3_3/src/main/java/ognl/Ognl.java#L246-L255), so this must be something else
```
    public static OgnlContext createDefaultContext(Object root, ClassResolver classResolver) {
        MemberAccess memberAccess = new AbstractMemberAccess() {
            @Override
            public boolean isAccessible(OgnlContext context, Object target, Member member, String propertyName) {
                int modifiers = member.getModifiers();
                return Modifier.isPublic(modifiers);
            }
        };
        return addDefaultContext(root, memberAccess, classResolver, null, new OgnlContext(classResolver, null, null));
    }
```

**Comment by oosato-im** [comment]
You are correct. the second parameter of `addDefaultContext` is passed the default `MemberAccess`.
However, null was passed to `OgnlContext`'s third parameter (`addDefaultContext`'s fifth parameter).
This cause throwing `IllegalArgumentException`.

`return addDefaultContext(root, memberAccess, classResolver, null, new OgnlContext(classResolver, null, `**null**`));`

**Comment by lukaszlenart** [comment]
Oh, right! Fixed in #174

**Comment by lukaszlenart** [comment]
OGNL 3.3.4 is under way to the Maven Central
